recipes/notebook-shim/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "notebook-shim" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/notebook_shim-{{ version }}.tar.gz
+  sha256: 7897e47a36d92248925a2143e3596f19c60597708f7bef50d81fcd31d7263e85
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - jupyter-packaging >=0.10,<1
+    - pip
+    - python >=3.7
+  run:
+    - jupyter_server >=1.8,<2
+    - python >=3.7
+
+test:
+  imports:
+    - notebook_shim
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/jupyterlab/notebook_shim
+  summary: A shim layer for notebook traits and config
+  license: BSD-3-Clause
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - jtpio
+    - Zsailer
```

Comment:
cc @Zsailer if you are fine being listed here. Thanks!

Reply:
:+1: